src/Samhammer.AzureBlobStorage/StreamHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Microsoft.IO;
+
+namespace Samhammer.AzureBlobStorage
+{
+    public class StreamHelper
+    {
+        private const long DefaultMaxSmallPoolFreeBytes = 1000000;
+        private const long DefaultMaxLargePoolFreeBytes = 10000000;
+        private static string maxSmallPoolFreeBytes = Environment.GetEnvironmentVariable("MAX_SMALLPOOL_FREEBYTES");
+        private static string maxLargePoolFreeBytes = Environment.GetEnvironmentVariable("MAX_LARGEPOOL_FREEBYTES");
+        public static readonly RecyclableMemoryStreamManager StreamManager = new RecyclableMemoryStreamManager(
```

Comment:
You may need to add the stream manager into ioc context as singleton, cause you cant access options otherwise.

Reply:
It's great, Roli, I have updated it